docker-compose.yml

```diff
@@ -21,7 +21,9 @@ services:
     depends_on:
     - database
   scrape:
-    image: openstates/openstates-pupa
+    build:
```

Comment:
This and the requirements-new.txt changes were for testing purposes. The image isn't up to date yet, but now that spatula 0.0.2 is out, the next commit to this repository should fix that. I'll back this out in anticipation of the next automated build.